app/src/main/res/layout/item_add_on_pledge.xml

```diff
@@ -33,38 +33,77 @@
+                    style="=@style/Title2Medium"
+                    app:layout_constraintStart_toEndOf="@id/add_on_minimum"
+                    app:layout_constraintTop_toBottomOf="@id/title_container"
+                    tools:text=" + $5 shipping each"
+                    android:visibility="gone"
+                    android:textSize="@dimen/subheadline"
+                    android:textColor="@color/ksr_green_500"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"/>
+
+                <TextView
+                    android:id="@+id/add_on_conversion"
                     style="@style/FootnotePrimaryMedium"
+                    android:textColor="@color/ksr_dark_grey_500"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_marginTop="@dimen/grid_1_half"
                     app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/add_on_minimum_text_view"
+                    app:layout_constraintTop_toBottomOf="@id/add_on_minimum"
                     tools:text="About $15 USD"
                     tools:visibility="visible"/>
 
-                <include
-                    android:id="@+id/title_container"
-                    layout="@layout/add_on_title"
+                <TextView
+                    android:id="@+id/add_on_description"
+                    style="@style/BodyPrimary"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:visibility="visible"
-                    android:layout_marginTop="@dimen/grid_2"
-                    app:layout_constraintTop_toBottomOf="@id/add_on_conversion_text_view"
+                    android:layout_marginTop="@dimen/grid_3"
+                    android:layout_marginBottom="@dimen/grid_2"
+                    app:layout_constraintRight_toRightOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintEnd_toEndOf="parent" />
+                    app:layout_constraintTop_toBottomOf="@id/add_on_conversion"
+                    tools:text="@string/Pledge_any_amount_to_help_bring_this_project_to_life" />
+
+                <View
+                    android:id="@+id/divider"
+                    android:layout_width="match_parent"
+                    android:layout_marginTop="18dp"
+                    android:layout_height="1dp"
+                    android:background="@drawable/divider_dark_grey_500_horizontal"
+                    android:importantForAccessibility="no"
+                    app:layout_constraintTop_toBottomOf="@+id/add_on_description"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent" />
+
+
 
                 <include
```

Comment:
I think there is a missing output logic, for the items.In case the reward has no items we should hide the container alongside with the "includes" title. Forget this comment if that logic is happening, is just I haven't seeing it 
![Screen Shot 2020-07-22 at 12 32 33 PM](https://user-images.githubusercontent.com/4083656/88220196-82438a80-cc17-11ea-984b-e2e30fd3b9c3.png)

Reply:
Great catch. Fixed!